media/css/quantum/_common.scss

```diff
@@ -0,0 +1,165 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+@import 'lib';
+
+
+$color-quantum-blue: #0a84ff;
+
+/* -------------------------------------------------------------------------- */
+// Common elements
+
+body {
+    @include open-sans;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-weight: normal;
+}
+
+/* -------------------------------------------------------------------------- */
+// Key features sections
+
+.key-features-section {
```

Comment:
Would this be worth putting in it's own file to import? I'm curious how regularly this will be used, or if pages would just end up importing unused CSS.

Reply:
Will refactor into its own file.